bindings/nodejs/lib/types/block/basic_block.ts

```diff
@@ -1,22 +1,30 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import type { HexEncodedString } from '../utils/hex-encoding';
 import { Payload, PayloadDiscriminator } from './payload';
 import { Type } from 'class-transformer';
+import { StrongParents, WeakParents, ShallowLikeParents } from './parents';
 
 /**
- * Block layout.
+ * Basic Block layout.
  */
 export class Block {
```

Comment:
What about ValidationBlock?

Reply:
Different issue

Reply:
Sure, but shouldn't this structure now be a `BasicBlock` at least?

Reply:
ye i suppose so, 1 sec

Reply:
But isn't this rather the BlockWrapper that's shared between both block types?